sources/packages/backend/apps/db-migrations/src/migrations/1707343675759-UpdateProgramCompletionYears.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class UpdateProgramCompletionYears1707343675759
```

Comment:
I'm not sure why we need to update the database migration for this PR. Can you briefly explain?

Reply:
I believe he did it for the existing column completion_years value created in the EducationPrograms table from 1YearToLessThan2Years to 60WeeksToLessThan2Years

Reply:
Thanks @guru-aot. Correct me if I'm wrong. I think PT applications will be affected with this migration, and we probably don't want it since the changes in this PR is only for FT.

Reply:
Just want to keep a record here. The database migration will affect both FT and PT applications. However, since PT calculations in Camunda do not take `program length` into consideration, this update doesn't affect PT calculations. This database migration is a good approach for this PR. Conversation resolved.